applications/circlemaker/doc/index.md

```diff
@@ -0,0 +1,148 @@
+/*                                                                                                                                                                                                                                                                                                 
+Section: Circlemaker
+Title: Circlemaker
+Language: en-US
+*/
+
+# Overview
```

Comment:
A list of TODOs was added instead of existing text, and it will be filled later.